### PR TITLE
JAMES-2773 Fix CI benchmark glowroot creation

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: JenkinsfileStressTests [hunk 1]
```diff
@@ -1,88 +0,0 @@
-// This file should respect the Jenkinsfile format describe here:
```
Comment (trantienduchn) [inline JenkinsfileStressTests]: why did you delete this file?

**Comment by blackheaven** [comment]
test this please

**Comment by blackheaven** [comment]
test this please
